### PR TITLE
Fix Issue With Bundle Images

### LINKED ISSUE #949: [WPF] Loading Images From Bundle
Loading images works fine in development but when you do a bundle the javascript tries to load images from `ms-appx:///` (UWP). Theoretically, as far as I can tell, the correct WPF source is `pack://application,,,/` however we were unable to get it working.

### PR DESCRIPTION
**Pull request by kevinvangelder**
Fixes #949

### CODE REVIEW

**Comment by rozele** [comment]
The ms-appx:/// prefix comes from the fact that you set your bundle path to start with ms-appx:///, i.e. [https://github.com/ReactWindows/react-native-windows/blob/master/ReactWindows/Playground.Net46/AppReactPage.cs#L19](https://github.com/ReactWindows/react-native-windows/blob/master/ReactWindows/Playground.Net46/AppReactPage.cs#L19).  If you simply change that to use your own conventions for WPF, all the rest of the relative URIs will be adjusted accordingly.


---
In reply to: [270760015](https://github.com/ReactWindows/react-native-windows/pull/950#issuecomment-270760015) [](ancestors = 270760015)

**Comment by rozele** [comment]
:clock1:

File: ReactWindows/ReactNative.Net46/Views/Image/ReactImageManager.cs [hunk 1]
```diff
@@ -306,7 +306,7 @@ private void OnImageStatusUpdate(Border view, ImageStatusEventData status)
                     status => OnImageStatusUpdate(view, status),
                     _ => OnImageFailed(view));
 
-                image.UriSource = new Uri(source);
+                image.UriSource = new Uri(source, UriKind.Absolute);
```
Comment (rozele) [inline ReactWindows/ReactNative.Net46/Views/Image/ReactImageManager.cs]: >UriKind [](start = 50, length = 7)

Do you need to qualify this Absolute here?  Doesn't the Uri constructor handle this by default? [https://referencesource.microsoft.com/#System/net/System/URI.cs,399](https://referencesource.microsoft.com/#System/net/System/URI.cs,399)

File: ReactWindows/ReactNative.Shared/Bridge/JavaScriptBundleLoader.cs [hunk 2]
```diff
@@ -100,7 +100,7 @@ public override Task InitializeAsync()
                 var assembly = Assembly.GetAssembly(typeof(JavaScriptBundleLoader));
                 var assemblyName = assembly.GetName();
                 var pathToAssembly = Path.GetDirectoryName(assemblyName.CodeBase);
-                var pathToAssemblyResource = Path.Combine(pathToAssembly, SourceUrl.Replace("ms-appx:///", String.Empty));
+                var pathToAssemblyResource = Path.Combine(pathToAssembly, SourceUrl);
                 var u = new Uri(pathToAssemblyResource);
```
Comment (rozele) [inline ReactWindows/ReactNative.Shared/Bridge/JavaScriptBundleLoader.cs]: Why do you still need to do a path.Combine here?  Isn't the SourceUrl now based on the initial bundle URL?

File: local-cli/generator-wpf/templates/src/AppReactPage.cs [hunk 3]
```diff
@@ -14,7 +15,7 @@ public override string JavaScriptBundleFile
         {
             get
             {
-                return "ms-appx:///ReactAssets/index.windows.bundle";
+                return AppDomain.CurrentDomain.BaseDirectory + "ReactAssets/index.windows.bundle";
```
Comment (matthargett) [inline local-cli/generator-wpf/templates/src/AppReactPage.cs]: can this be made to match the inline getter syntax in Playground (or vice versa)?

Comment (kevinvangelder) [inline local-cli/generator-wpf/templates/src/AppReactPage.cs]: Yes, I meant to do that before but forgot.

**Pull request merged**